pkg/ytconfig/node.go

```diff
@@ -317,7 +362,7 @@ func getExecNodeLogging(spec *ytv1.ExecNodesSpec) Logging {
 		[]ytv1.TextLoggerSpec{defaultInfoLoggerSpec(), defaultStderrLoggerSpec()})
 }
 
-func getExecNodeServerCarcass(spec *ytv1.ExecNodesSpec, usePorto bool) (ExecNodeServer, error) {
+func getExecNodeServerCarcass(spec *ytv1.ExecNodesSpec, ytsaurusSpec *ytv1.YtsaurusSpec) (ExecNodeServer, error) {
```

Comment:
This change would conflict with #75 feature, in which i've made (not yet merged) some refactorings to be able to use code (in particular config generator) without having `YtsaurusSpec` (because remote nodes won't have one). 
Could you please not add ytsaurusSpec parameter and just pass needed arguments as is (and maybe in future they can be combine them in some struct if needed).